src/main/java/frc/robot/commands/CurvatureDriveCommand.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.operatorInputs.Controls;
+import frc.robot.subsystems.drive.Drive;
+
+public class CurvatureDriveCommand extends CommandBase {
+
+    //classes
+    private Drive drive;
+    private Controls controls;
+
+    //constants
+    private final double DEADZONE = 0.2;
+    private final double TURN_CONSTANT = 0.5;
+  
+    public CurvatureDriveCommand(Controls controls, Drive drive) {
+        this.controls = controls;
+        this.drive = drive;
+
+        addRequirements(drive);
+    }
+
+    // Called just before this Command runs the first time
+    @Override
+    public void initialize() {
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    @Override
+    public void execute() {
+           /* double throttle = controls.getLeftY(DEADZONE);
```

Comment:
All this commented out code needs to be reenabled or we won't drive

Reply:
done